### DISCUSSION TITLE
Can I use this in a panel on a windows forms app?

### DISCUSSION THREAD

**Discussion by shawty** [spectreconsole/spectre.console#1150] [Q&A] [answered]
I have a need to provide an in-app logging console, on a regular windows forms "FORM" (With other components), such that this looks more like a scrolling text box, but with the traditional coloured console output and font style of text.

Can I use spectre in this manner?

Or is it only useable in it's own console style window.

**Comment by patriksvensson** [comment, marked as answer]
Spectre.Console outputs VT-100 escape codes for styling, so you should be fine if you have a textbox or similar that can render that. We do not provide anything like that within the project though.

**Reply by shawty** [reply]
ok, thanks for the confirmation :-)

No I don't have an existing comp that handles VT-100 sadly.  Looks like I'll have to write my own :-)

Cheers
Shawty
